gooey/image_cache.lua

```diff
@@ -15,7 +15,7 @@ end
 -- @param filename
 -- @return Save file path
 local function get_save_file_path(filename)
-	local path = sys.get_save_file(fix(sys.get_config("project.title"):gsub(" ", "_")), filename)
```

Comment:
Interesting. That `gsub(" ", "_")` seems pointless since `fix()` already replaces spaces with `+`. I wonder why I haven't spotted this before... 🤷